.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: build
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release/*'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - '*'
+
+env:
+  DOCKERHUB_SLUG: distribution/distribution
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ${{ env.DOCKERHUB_SLUG }}
+          ### versioning strategy
+          ### push semver tag v2.9.0 on main (default branch)
+          # distribution/distribution:2.9.0
+          # distribution/distribution:latest
+          ### push semver tag v2.8.0 on release/2.8 branch
+          # distribution/distribution:2.8.0
+          ### push on main
+          # distribution/distribution:edge
+          tags: |
+            type=semver,pattern={{version}}
+            type=ref,event=pr
+            type=edge
```

Comment:
shouldn't this be `type={{version}}-edge`?  This workflow should only be pushing `2.x` image tags

Reply:
`type=edge` will not be pushed for `release/*` because it's only trigerred on default branch (main): https://github.com/docker/metadata-action#typeedge

Reply:
Yeah, I think we should remove that 🤔  again, for the same reasons as my previous comment 😅

Reply:
Sure done